testing/content/files/en-us/markdown/index.md

```diff
@@ -24,6 +24,9 @@ A bullet point list:
 
 This paragraph as a [link](/en-US/docs/Web).
 
+<pre class="brush: html notranslate"><code><span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>pre</span><span class="token punctuation">&gt;</span></span><span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>code</span><span class="token punctuation">&gt;</span></span>code
+</code></pre>
+
```

Comment:
Awesome! I was going to ask; how do I test this.
But why would this even exist? ...in Markdown. Is it going to be triple-backtick-html instead of a `<pre>` tag?

Reply:
We will mostly get rid of the problem, but there are a couple of <pre> tags that the converter is not touching, like this one: https://github.com/mdn/content/blob/h2m-js/files/en-us/web/javascript/equality_comparisons_and_sameness/index.md#when-to-use-objectis-versus-triple-equals
In that case it is because there are `<code>` tags inside of it.